cabal-install/tests/UnitTests/Distribution/Client/ProjectConfig.hs

```diff
@@ -654,7 +657,7 @@ instance Arbitrary PackageConfig where
                       , packageConfigHaddockContents = x40'
                       , packageConfigHaddockForHackage = x41' }
       |  (((x00', x01', x02', x03', x04'),
-          (x05', x42', x06', x07', x08', x09'),
+          (x05', x42', x06', x44', x07', x08', x09'),
```

Comment:
What on earth was the author of the surrounding lines thinking?

The above `arbitrary <*> arbitrary <*> arbitrary` on 46 positional arguments is already contributor unfriendly but this one really sets a new bar.